src/views/templates/details/tabs/disks/TemplateDisksPage.tsx

```diff
@@ -71,27 +71,27 @@ const TemplateDisksPage: FC<TemplateDisksPageProps> = ({ obj: template }) => {
     <div className="template-disk-tab">
       <ListPageBody>
         <SidebarEditor<V1Template> resource={template} onResourceUpdate={onSubmitTemplate}>
-          <Flex>
-            <FlexItem>
-              <ListPageCreateButton
-                className="list-page-create-button-margin"
-                isDisabled={isEditDisabled}
-                onClick={() =>
-                  createModal(({ isOpen, onClose }) => (
-                    <DiskModal
-                      vm={vm}
-                      isOpen={isOpen}
-                      onClose={onClose}
-                      onSubmit={onUpdate}
-                      headerText={t('Add disk')}
-                      createOwnerReference={false}
-                    />
-                  ))
-                }
-              >
-                {t('Add disk')}
-              </ListPageCreateButton>
-            </FlexItem>
+          <Flex className="list-page-create-button-margin">
+            {isTemplateEditable && (
+              <FlexItem>
+                <ListPageCreateButton
```

Comment:
can you use the `createAccessReview` prop instead of using the hook?

Reply:
`createAccessReview` will remove the button. We decided to not remove the button but just disable it. I'll update this pr again